lvmetrics/node_updator.go

```diff
@@ -110,3 +114,23 @@ func (n *NodePatcher) addFinalizer(node *corev1.Node) {
 	}
 	node.Finalizers = append(node.Finalizers, topolvm.NodeFinalizer)
 }
+
+// GetCapacity gets available bytes of VG from the node annotation
```

Comment:
This should not be necessary because at startup `lvmd` replies the current
free space capacity as soon as possible.